docassemble/MAVirtualCourt/data/questions/209A_affidavit.yml

```diff
@@ -0,0 +1,499 @@
+   
+   We have to check at least one box to get a restraining order.
+   
+   Use the green button to help you decide the boxes you need to check.
+fields:
+  - ${ defendant } made me AFRAID that ${ defendant }  was going to seriously hurt me, physically.: incidents_of_abuse[i].abuse.feared_harm
+    datatype: yesnowide
+  - ${ defendant } tried to physically hurt me.: incidents_of_abuse[i].abuse.defendant_abuse_attempted_physical_harm
+    datatype: yesnowide
+  - ${ defendant } physically hurt me.: incidents_of_abuse[i].abuse.defendant_abuse_caused_physical_harm
+    datatype: yesnowide
+  - ${ defendant }  threatened, pressured or forced me to have sex with ${ defendant }.: incidents_of_abuse[i].abuse.defendant_abuse_caused_sexual_assault
+    datatype: yesnowide
+help:
+  label: Can you give me some examples?
+  heading: Can you give me some examples?
+  content: |
+    Making you afraid of physical harm includes saying things like "I'll kill you if you leave me," or "If you don't give me the car keys, I'll hit you harder than last time." It can also include waving a gun or knife at you, smashing your furniture, driving recklessly, and hurting or killing your pets.It includes threatening to commit any act of violence, threatening with a fake gun, or bomb threats. 
+    
+    slamming fist into hand, into wall, blocking not letting you get by, standing between you and the door, not letting you get back.... 
+    
+    Looming over you, menacing..Spitting on you, punching %%HISHER([Defendant gender MC])%% fist into %%HISHER([Defendant gender MC])%% hand, putting a fist through the wall, 
+    More - See a list of examples on MassLegalHelp.... 
+    
+    DV Advocates contributed and refined this help screen and the list on MLH. 
+    
+    Physically hurting you includes shoving, pushing, punching, slapping, kicking, pulling your hair, choking, strangling, throwing things at you, burning you with hot food or cigarettes, stabbing, shooting, or locking you in a room.
+    Threatening, pressuring or forcing you to have sex also includes forced contact with intimate body parts. 
+    For children under 18, it can mean any sexual contact.
+---
+id: Affidavit description
+question: |
+  Describe the incident  
+subquestion: |
+   In your own words, what happened? Please use complete sentences, and be as descriptive as you can. List as many details and witnesses as you can remember. 
+   
+   See some examples
+fields: 
+  - no label: incidents_of_abuse[i].description
+    input type: area
+    default: |
+      On or about ${ incidents_of_abuse[i].date }, ${ incidents_of_abuse[i].time_of_day } at ${ incidents_of_abuse[i].location } 
+help:
+  label: How descriptive should I be?
+  heading: How descriptive should I be?
+  content: |
+    2 examples to help you understand how to write a good description: 
+    
+    **A poor example:** 
+    
+    We were arguing. Defandant screamed at me and I was afraid. 
```

Comment:
Do we want this to use the abuser's name, or is that too triggering?

Reply:
Yes absolutely - we want to use the Defendant's first name every time we refer to them throughout the entire 209A interview.